netmiko/optilink/op_golt944.py

```diff
@@ -0,0 +1,13 @@
+from netmiko.huawei.huawei import HuaweiTelnet
+
+
+class OptilinkGOLT944Telnet(HuaweiTelnet):
+    """Optilink GOLT 944 telnet driver"""
+
+    def session_preparation(self) -> None:
+        self.ansi_escape_codes = True
+        self._test_channel_read(pattern=r"[>\]]")
+        self.set_base_prompt()
+        self.config_mode()
+        self.disable_paging(command="screen-rows per-page 0")
```

Comment:
@bsdemon Are you sure that is the right command to disable output paging? This is what you would typically use on a Ciena SAOS and not on a Huawei based device?

Reply:
Yes, it is.